.github/workflows/updates-testing-nightly.yml

```diff
@@ -0,0 +1,19 @@
+name: fedora-updates-testing
+on:
+  schedule:
+    - cron: '0 1 * * *'
+  # can be run manually on https://github.com/cockpit-project/cockpit-podman/actions
+  workflow_dispatch:
```

Comment:
this needs to add `permissions:` By default, the token only has read-only permissions, I figure it needs write access to at least "statuses: write"